projects/clinician-insights/daily-feedback.py

```diff
@@ -0,0 +1,413 @@
+
+    reponse1, reponse2 = \
+        get_json_data(args.email, args.password, userID, outputFileLocation, startDate, endDate)
+
+    metaData.loc[dIndex, ["getData.response1", "getData.response2"]] = \
+        reponse1.status_code, reponse2.status_code
+
+    # load json data
+    data = pd.read_json(outputFileLocation)
+
+    if "type" in list(data):
+        if "cbg" in data.type.unique():
+
+            # calculate stats
+            cgmData = data[data.type == "cbg"].copy()
+            cgmData["utcTime"] = pd.to_datetime(cgmData.time, utc=True)
+
+            # get data from 6am to 6am
+            if (("timezone" in list(data)) | ("timezoneOffset" in list(data))):
+                if "timezone" in list(data):
+                    userTz = data.timezone.describe()["top"]
+                    tzo = get_timeZoneOffset(reportDate, userTz)
+                    tz = timezone(userTz)
+
+                    start6amDate = tz.localize(pd.to_datetime(reportDate)
+                                               - pd.Timedelta(1, unit="D")
+                                               + pd.Timedelta(5, unit="h")
+                                               + pd.Timedelta(57, unit="m")
+                                               + pd.Timedelta(30, unit="s"))
+
+                    end6amDate = tz.localize(pd.to_datetime(reportDate)
+                                             + pd.Timedelta(5, unit="h")
+                                             + pd.Timedelta(57, unit="m")
+                                             + pd.Timedelta(30, unit="s"))
+
+                    cgm = cgmData.loc[((cgmData.utcTime > start6amDate) &
+                                       (cgmData.utcTime < end6amDate)), ["time", "value"]]
+
+                else:  # if there is no timezone given, then infer from timezone offset
+                    tzo = data.timezoneOffset.median()
+                    start6amDate = (pd.to_datetime(reportDate)
+                                    - pd.Timedelta(1, unit="D")
+                                    + pd.Timedelta(5, unit="h")
+                                    + pd.Timedelta(57, unit="m")
+                                    + pd.Timedelta(30, unit="s")
+                                    - pd.Timedelta(tzo, unit="m"))
+
+                    end6amDate = (pd.to_datetime(reportDate)
+                                  + pd.Timedelta(5, unit="h")
+                                  + pd.Timedelta(57, unit="m")
+                                  + pd.Timedelta(30, unit="s")
+                                  - pd.Timedelta(tzo, unit="m"))
+
+                    cgm = cgmData.loc[((pd.to_datetime(cgmData.time) > start6amDate) &
+                                       (pd.to_datetime(cgmData.time) < end6amDate)), ["time", "value"]]
+
+                cgm = cgm.rename(columns={"value": "mmol_L"})
+                cgm["mg_dL"] = (cgm["mmol_L"] * 18.01559).astype(int)
```

Comment:
Add 18.01559 as a constant in `tidals`?

Reply:
@pazaan YES! @jameno and I will be moving a lot code over to tidals...it just might take a little bit of time to do so:)